.github/workflows/kanister-image-build.yaml

```diff
@@ -7,6 +7,14 @@ on:
         description: 'Image tag in the format vx.x.x'
         required: true
         type: string
+      latest_tag:
```

Comment:
this workflow is run automatically right?
how would someone decide if they want to tag the image to latest or something else?

Reply:
No, this pipeline does not run automatically. `workflow_dispatch` trigger can only be triggered manually and the inputs have to be set by the person triggering it.

- `dev_latest` should be set when building the test image (e.g. to test changes in the image)
- `latest` should be set when building the image to be used for downstream users, such as `run-container.sh` and CI pipelines